application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -57,6 +59,19 @@ public void onSlashCommand(@NotNull final SlashCommandEvent event) {
         String userID = (Objects.requireNonNull(event.getMember()).getId());
         TeXFormula formula;
         try {
+            String regex = "(\\$\\S+\\$)|([a-zA-Z!., ]+)";
+            Pattern pattern = Pattern.compile(regex);
+            Matcher matcher = pattern.matcher(latex);
+            StringBuilder sb = new StringBuilder();
+            while (matcher.find()) {
+                if (matcher.group(1) != null) {
+                    sb.append(matcher.group(1).replaceAll("\\$", ""));
```

Comment:
Use `.replace("$", "")`, not `replaceAll` (which is for regex, that u dont need here)